src/sdkV2.ts

```diff
@@ -183,6 +197,7 @@ export default class CegaEvmSDKV2 {
       throw new Error('Signer not defined');
     }
     const proxyEntry = await this.loadCegaWrappingProxy();
+    // TODO: update also with dcs prefix?
```

Comment:
remove this todo

decided not to rename the SC function anymore. it's called on proxy and not on CegaEntry so won't have naming collisions in the future

and want to minimize diffs between SC test deployment and final production now